homeworks/01-bubble-sort/src/components/buttons.tsx

```diff
@@ -0,0 +1,11 @@
+import React from "react";
+
+type ButtonNewSetProps = {
+    setBubblesFunction: () => void;
+    sortInProgress: boolean,
+    buttonName: string,
+}
+
+export const ControlButton: React.FC<ButtonNewSetProps> = ({ setBubblesFunction, sortInProgress, buttonName }) => {
+    return <button onClick={() => setBubblesFunction()} disabled={sortInProgress}>{buttonName}</button>;
```

Comment:
Don't need to create extra function every render: `onClick={setBubblesFunction}`